### DISCUSSION TITLE
Is there a way to change annotate.draw_text position other then center?

### DISCUSSION THREAD

**Discussion by shin616122** [roboflow/supervision#243] [Q&A] [answered]
Sorry, I am new to Python and Roboflow.
I used the code below that I found in one of the example projects to show the tracking object counts but I want to have the label to show in a different position other than the center.
Is it possible? 

```
zone_polygon = (ZONE_POLYGON * np.array([frame_width, frame_height])).astype(int)
zone = sv.PolygonZone(polygon=zone_polygon, frame_resolution_wh=tuple([frame_width, frame_height]))
zone_annotator = sv.PolygonZoneAnnotator(
    zone=zone,
    color=sv.Color.red(),
    thickness=0,
    text_thickness=4,
    text_scale=2,
)

zone.trigger(detections=detections)
frame = zone_annotator.annotate(scene=frame)
```

**Comment by hardikdava** [comment, marked as answer]
hello @shin616122 👋 , thanks for taking interest in `supervision`. We are already planning for such feature. Also we are working on different `Annotators` which will be released soon. Stay updated for new features.

@SkalskiP , we already talked about similar feature.

**Reply by shin616122** [reply]
Ok understood
Thank you for the respone

**Comment by blackksheep10** [comment]
@hardikdava So. what about now, this feature is available now??

**Reply by SkalskiP** [reply]
@blackksheep10, how would you like to position the text?

**Reply by blackksheep10** [reply]
The position of counting text isn't center of polygonZoneAnnotator

**Reply by SkalskiP** [reply]
cool I got this, but where would you like it to be positioned?

**Reply by blackksheep10** [reply]
Left-top on the screen,  Could u do that  ?  Cool

**Reply by SkalskiP** [reply]
It is not implemented directly, but you can still do it quickly. You need to skip the annotator and do it yourself.

- This part is responsible for loading the image, defining the zone, running detection and triggering the zone.

  ```python
  import cv2
  import numpy as np
  import supervision as sv
  from ultralytics import YOLO
  
  # define polygon vertices
  POLYGON = np.array([
      [100, 100],
      [300, 100],
      [300, 300],
      [100, 300]
  ])
  
  # load model
  model = YOLO('yolov8s.pt')
  
  # read image
  image = cv2.imread(...)
  
  # get loaded image resolution
  height, width = image.shape[:2]
  resolution_wh = (width, height)
  
  # initiate polygon zone
  polygon_zone = sv.PolygonZone(polygon=POLYGON, resolution_wh=resolution_wh)
  
  # run model inference
  result = model(image)[0]
  detections = sv.Detections.from_ultralytics(result)
  
  # trigger zone
  polygon_zone.trigger(detections=detections)
  ```

- This part is responsible for drawing the zone and the counter.

  ```python
  # x = 50, y = 50 is the center of your counter
  text_anchor = sv.Point(x=50, 50)
  
  annotated_image = image.copy()
  
  # draw polygon zone
  annotated_image = sv.annotated_frame = sv.draw_polygon(
      scene=annotated_image,
      polygon=POLYGON,
      color=sv.Color.red()
  )
  
  # draw counter
  annotated_frame = sv.draw_text(
      scene=annotated_frame,
      text=str(polygon_zone.current_count),
      text_anchor=text_anchor,
      background_color=sv.Color.red(),
      text_color=sv.Color.white()
  )
  ```

**Reply by blackksheep10** [reply]
Thank you a lot bro, I will try it 🙏

**Reply by SkalskiP** [reply]
Sure! Let me know how it went!

**Reply by blackksheep10** [reply]
![Screenshot 2024-01-17 223157](https://github.com/roboflow/supervision/assets/128966434/a89465e4-39b5-4fa6-bbfe-5167e50661e9)
 why draw_polygon() is not defined ??

**Reply by SkalskiP** [reply]
It should be `sv.draw_polygon`
